src/Template/User/profile.ctp

```diff
@@ -37,6 +37,10 @@ use App\Model\CurrentUser;
  * @link     https://tatoeba.org
  */
 
+$this->Html->script('user/language.ctrl.js', ['block' => 'scriptBottom']);
+$this->Html->script('/js/directives/sentence-and-translations.dir.js', array('block' => 'scriptBottom')); // TODO: this is just to get the languageIcon directive
```

Comment:
I think we could consider just moving the `languageIcon` directive to `app.module.js`. As we migrate more pages to AngularJS, this directive will be used in many more places so I think it would make sense to have it in there.

If not, you can extract the directive into its own file (`language-icon.dir.js`), then inject it in both `profile.ctp` and `sentence_and_translations.ctp`.

Or, after merging this into `dev`, you can include it in the `[sentence-component.js]` of the [asset_compress.ini](https://github.com/Tatoeba/tatoeba2/blob/dev/config/asset_compress.ini#L55]) instead of injecting it in `sentence_and_translations.ctp`.